flare_portal/experiments/models/data.py

```diff
@@ -127,5 +136,63 @@ class FearConditioningData(BaseData):
         "rating",
     ]
 
-    def __str__(self) -> str:
-        return f"Participant: {self.participant_id} - Module: {self.module_id}"
+
+class BasicInfoData(BaseData):
+    GENDERS = Choices(
+        ("male", "Male"),
+        ("female", "Female"),
+        ("non_binary", "Non-binary"),
+        ("self_define", "Prefer to self-define"),
+        ("dont_know", "Don't know"),
+        ("no_answer", "Prefer not to answer"),
+    )
+    HEADPHONE_TYPES = Choices(
+        ("in_ear", "In-ear"), ("on_ear", "On-ear"), ("over_ear", "Over-ear"),
```

Comment:
Could we use `IN_EAR`, `ON_EAR`, `OVER_EAR` for constants?

Reply:
Ugh I hate that it's not the same, however all the other choices in the portal code is already snake case, so having this one place where it's different will be weird.

I'm keeping it as is. I think it's normal that the interface code (API client) does these sorts of transformations anyway. I know this will be annoying for you so all I have to say is... sorry.